docs/blog/2024-community-update.rst

```diff
@@ -0,0 +1,84 @@
+.. post:: Aug 8, 2024
+   :tags: stats, newsletter
+   :author: Eric Holscher
+
+Write the Docs Community Update - August 2024
+=============================================
+
+Hey there documentarians!
+
+Eric Holscher here filling in for the Newsletter team as they take their normal August break.
+
+We're looking forward to seeing everyone online for our virtual `Atlantic <http://www.writethedocs.org/conf/atlantic/2024/>`_ on **September 22-24**. The event is scheduled to be available for both folks in EDT and CEST timezones. Tickets are still on sale until a few days before the event, so grab yours now if you haven't yet.
+
+We're also getting excited about our upcoming `Australia <http://www.writethedocs.org/conf/australia/2024/>`_ conference on **November 28-29** in Melbourne. The `Australia conference CFP <https://www.writethedocs.org/conf/australia/2024/cfp/>`_ is open for a few more weeks if you have ideas, and can make it to Melbourne.
+
+We held our second Portland conference in our new venue this year, and it felt like everything worked a little bit better than last year. You could really feel the improvement that another year of iterating on the event brought.  The `photos <https://www.flickr.com/photos/writethedocs/albums/72177720316224454/>`_ and `videos <https://www.youtube.com/playlist?list=PLZAeFn6dfHpm4FboYSTD1Bs8Yp8k_JvAL>`_ from the conference are available.
+
+2025 Conferences
+----------------
+
+We are still in the planning phase for our 2025 conferences, but expect the structure to remain similar. Our Portland conference will be **May 4-6, 2025**, and expect more updates about the other events as we get closer to the end of the year.
+
+Kenya Event and Community Engagement
+------------------------------------
+
+We are thrilled to work with the Meetup team in Kenya to put on `a small gathering for their community <https://wtdkenya.hashnode.dev/write-the-docs-kenya-summit-2024>`_ this year! It was a great way to continue building the community locally, and we're excited to see more events like this in the future.
+
+If you're interested in organizing a community event, please feel free to reach out to the core team. We have resources and support to help you get started on a Meetup, or to grow your meetup into a larger community event.
+As part of this effort, we're going to be building on our Grant program for conference to extend it to other types of events.
+Look for more updates on that in the coming months.
+
+Job Board Transition
+--------------------
+
+We are looking at transitioning away from a job board and to a job sponsorship model that works similarly to our other `community sponsorship options <https://www.writethedocs.org/sponsorship/>`_. Our current job board vendor has raised their rates significantly, which caused us to reevaluate how to promote jobs in the community. Our Slack ``#job-posts-only`` channel has quite active with job postings, and we're looking to expand on that success.
```

Comment:
```suggestion
We are looking at transitioning away from a job board and to a job sponsorship model that works similarly to our other `community sponsorship options <https://www.writethedocs.org/sponsorship/>`_. We have been reevaluating how to promote jobs in the community, and felt the job board was no longer the best fit. Our Slack ``#job-posts-only`` channel has quite active with job postings, and we're looking to expand on that success.
```
We had doubts about the job board even before the price change - this feels more positive to me. We're doing this because the transition is a positive, not just because we want to save money.